src/rx5808.cpp

```diff
@@ -18,12 +32,19 @@ RX5808::Init()
         digitalWrite(PIN_CS_2, HIGH);
     #endif
 
+    SPIModeEnabled = true;
+
     DBGLN("SPI config complete");
 }
 
 void
 RX5808::SendIndexCmd(uint8_t index)
 {
+    if (!SPIModeEnabled) 
```

Comment:
I think this guard should be moved to the read/write register functions.  Just in case other commands are added and SendIndexCmd() isnt called.

Reply:
The SPIEnable guard has been moved to the read and write functions.